velox/exec/AggregateWindow.cpp

```diff
@@ -374,6 +408,10 @@ class AggregateWindowFunction : public exec::WindowFunction {
   // Stores metadata about the previous output block of the partition
   // to optimize aggregate computation and reading argument vectors.
   std::optional<FrameMetadata> previousFrameMetadata_;
+
+  // Stores the initialized value of the aggregate for empty input. This value
```

Comment:
Rephrase "Stores default result value for empty frame aggregation. Window functions return the default value of an aggregate (aggregation with no rows) for empty frames. e.g. count for empty frames should return 0 and not null."

Reply:
Updated, thanks.